CSC8503CoreClasses/GuardObject.cpp

```diff
@@ -93,8 +98,22 @@ void GuardObject::HandleAppliedBuffs(float dt) {
 	}
 }
 
-void GuardObject::MoveTowardFocalPoint(Vector3 direction) {
-	Vector3 dirNorm = direction.Normalised();
+bool GuardObject::CheckPolyDistance() {
+	if (mDist < 36) {
```

Comment:
Nice to have: rather than using a magic number convert it to a constant variable